src/pages/registration/RegistrationForm.tsx

```diff
@@ -158,7 +158,7 @@ const RegistrationForm = <T extends TissueValues<B>, B>({
                     return (
                       <motion.div
                         ref={blockIndex === values.tissues[currentIndex].blocks.length - 1 ? lastBlockRef : null}
-                        key={block.clientId}
+                        key={blockIndex}
```

Comment:
better option for key than an index?

Reply:
Unfortunately, in this context there is not a key identifier possible, all the fields contained in blocks can repeat including clientId

Reply:
Ah ok, thats fine then